tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -419,13 +419,20 @@ function kbs_k8s_deploy() {
 	fi
 }
 
-# Return the kbs service host name in case ingress is configured
+# Return the kbs service public IP in case ingress is configured
```

Comment:
nit: Should we make this "echos the..." as (unfortunately) we can't return a string value here?